src/transaction/log_impl.h

```diff
@@ -1012,14 +1006,8 @@ extern int logtb_delete_global_unique_stats (THREAD_ENTRY * thread_p, BTID * bti
 extern int logtb_reflect_global_unique_stats_to_btree (THREAD_ENTRY * thread_p);
 extern int logtb_tran_update_all_global_unique_stats (THREAD_ENTRY * thread_p);
 
-extern int log_rv_undoredo_record_partial_changes (THREAD_ENTRY * thread_p, char *rcv_data, int rcv_data_length,
-						   RECDES * record, bool is_undo);
-extern int log_rv_redo_record_modify (THREAD_ENTRY * thread_p, LOG_RCV * rcv);
-extern int log_rv_undo_record_modify (THREAD_ENTRY * thread_p, LOG_RCV * rcv);
-extern char *log_rv_pack_redo_record_changes (char *ptr, int offset_to_data, int old_data_size, int new_data_size,
-					      char *new_data);
-extern char *log_rv_pack_undo_record_changes (char *ptr, int offset_to_data, int old_data_size, int new_data_size,
-					      char *old_data);
+extern int log_rv_redo_record_modify (THREAD_ENTRY * thread_p, LOG_RCV * rcv); // TODO: this and dependent function not used at all
```

Comment:
Move these declarations to log_recovery.h too. They are not used yet, but they are generic functions that may be used in the future for adding new recovery operations.

Reply:
Done